.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env",
```

Comment:
We just recently moved away from babel in favor of just typescript. See related https://github.com/CanopyTax/single-spa-angular/issues/16.

It looks like this `.babelrc` file isn't being used -- let's remove it

Reply:
My bad - or rather SourceTree's bad: seems like "use theirs" conflict resolution gets confused when "theirs" no longer contains the file.